src/libecalc/presentation/yaml/yaml_types/emitters/yaml_venting_emitter.py

```diff
@@ -222,7 +223,9 @@ def get_emissions(
                 .tolist()
             )
             emission_rate = [oil_rate * factor for oil_rate, factor in zip(oil_rates, factors)]
```

Comment:
Should we ensure oil rate is in Sm3/d by converting?

Reply:
Done